zone/embparser.cpp

```diff
@@ -163,7 +163,8 @@ const char *QuestEventSubroutines[_LargestEventID] = {
 	"EVENT_TASK_BEFORE_UPDATE",
 	"EVENT_AA_BUY",
 	"EVENT_AA_GAIN",
-	"EVENT_PAYLOAD"
+	"EVENT_PAYLOAD",
+	"EVENT_LEVEL_DOWN"
   #ifdef BOTS
 	,
```

Comment:
nit: I'd remove this floating comma and just put it at the end of `EVENT_LEVEL_DOWN` (and probably one after `EVENT_SPELL_EFFECT_BUFF_TIC_BOT`).

Even if BOTS is undefined the comma at the end will make it cleaner when adding new events since only one new line will need added.

Also remove the indent from `#ifdef BOTS`

Reply:
Will fix this when at PC.